src/displayapp/screens/Twos.h

```diff
@@ -26,11 +26,14 @@ namespace Pinetime {
 
         lv_obj_t* scoreText;
         lv_obj_t* gridDisplay;
-        TwosTile grid[4][4];
+        static constexpr int nCols = 4;
+        static constexpr int nRows = 4;
+        static constexpr int nCells = nCols * nRows;
```

Comment:
It seems to me that a lot of the variables here are full (unsigned) ints, but they could just be (u)int8_ts.

Reply:
I've noticed that sometimes using smaller integers results in a larger binary size. These specifically don't make a difference, but if `tryMerge()` and `tryMove()` functions used uint8_t, the binary size would increase. This is why I'm not always sure which is best to use.

Reply:
Hmm interesting... Maybe with smaller integers the compiler isn't able to do as many optimisations? Anyway, if it increases the binary size, I don't see much benifit to using smaller integers.